nnvm/src/top/tensor/elemwise.cc

```diff
@@ -684,6 +738,14 @@ NNVM_REGISTER_ELEMWISE_REDUCE_OP(elemwise_sum)
 .describe(R"code(Adds all input arguments element-wise.
 
 )code"  NNVM_ADD_FILELINE)
+.set_attr<FTVMCompute>(
+  "FTVMCompute", [](const NodeAttrs& attrs,
+                    const Array<Tensor>& inputs,
+                    const Array<Tensor>& out_info) {
+    const ElementWiseReduceParam& param = nnvm::get<ElementWiseReduceParam>(attrs.parsed);
```

Comment:
Yes, I think that num_args cannot be dropped from elemwise_sum in nnvm. Because, num_args is used in [here](https://github.com/dmlc/tvm/blob/master/nnvm/src/top/elemwise_op_common.h#L313-L315).